core/src/main/java/com/graphhopper/routing/weighting/custom/CustomProfileConfig.java

```diff
@@ -18,6 +18,7 @@ public CustomProfileConfig(String name) {
 
     public CustomProfileConfig setCustomModel(CustomModel customModel) {
         getHints().putObject(CustomModel.KEY, customModel);
+        getHints().putObject("custom_model_file", "empty");
```

Comment:
This is a bit ugly. If we set the custom model via Java API I think it should not be required to set a file (?)

Reply:
Ah ok I see